influxql/INFLUXQL.md

```diff
@@ -653,15 +664,17 @@ SELECT mean(value) FROM cpu WHERE region = 'uswest' GROUP BY time(10m) fill(0);
 ```
 from_clause     = "FROM" measurements .
 
-group_by_clause = "GROUP BY" dimensions fill(<option>).
+group_by_clause = "GROUP BY" dimensions fill(fill_option).
+
+into_clause     = "INTO" measurement .
```

Comment:
Should be...
```
into_clause     = "INTO" measurement | back_ref .

back_ref         = ( policy_name ".:MEASUREMENT" ) |
                   ( db_name "." [ policy_name ] ".:MEASUREMENT" ) .
```
e.g.,
```
SELECT mean(value) INTO cpu_1h.:MEASUREMENT FROM /cpu.*/
```
This will select from all measurements beginning with `cpu` into the same measurement name in the `cpu_1h` retention policy.